python/lsst/cp/testing/isr.py

```diff
@@ -0,0 +1,303 @@
+
+from lsst.ip.isr import IsrTask, IsrTaskConfig
+
+
+class CptIsrTaskConnections(pipeBase.PipelineTaskConnections,
+                            dimensions=("instrument", "exposure", "detector")):
+    ccdExposure = cT.Input(
+        name="raw",
+        doc="Input exposure to process.",
+        storageClass="Exposure",
+        dimensions=["instrument", "exposure", "detector"],
+    )
+    camera = cT.PrerequisiteInput(
+        name="camera",
+        storageClass="Camera",
+        doc="Input camera to construct complete exposures.",
+        dimensions=["instrument"],
+        isCalibration=True,
+    )
+
+    crosstalk = cT.Input(
+        name="crosstalk",
+        doc="Input crosstalk object",
+        storageClass="CrosstalkCalib",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    crosstalkSources = cT.Input(
+        name="isrOverscanCorrected",
+        doc="Overscan corrected input images.",
+        storageClass="Exposure",
+        dimensions=["instrument", "exposure", "detector"],
+        deferLoad=True,
+        multiple=True,
+    )
+    bias = cT.Input(
+        name="bias",
+        doc="Input bias calibration.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    dark = cT.Input(
+        name='dark',
+        doc="Input dark calibration.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    flat = cT.Input(
+        name="flat",
+        doc="Input flat calibration.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "physical_filter", "detector"],
+        isCalibration=True,
+    )
+    ptc = cT.Input(
+        name="ptc",
+        doc="Input Photon Transfer Curve dataset",
+        storageClass="PhotonTransferCurveDataset",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+    fringes = cT.Input(
+        name="fringe",
+        doc="Input fringe calibration.",
+        storageClass="ExposureF",
+        dimensions=["instrument", "physical_filter", "detector"],
+        isCalibration=True,
+    )
+    strayLightData = cT.Input(
+        name='yBackground',
+        doc="Input stray light calibration.",
+        storageClass="StrayLightData",
+        dimensions=["instrument", "physical_filter", "detector"],
+        deferLoad=True,
+        isCalibration=True,
+    )
+    bfKernel = cT.Input(
+        name='bfKernel',
+        doc="Input brighter-fatter kernel.",
+        storageClass="NumpyArray",
+        dimensions=["instrument"],
+        isCalibration=True,
+    )
+    newBFKernel = cT.Input(
+        name='brighterFatterKernel',
+        doc="Newer complete kernel + gain solutions.",
+        storageClass="BrighterFatterKernel",
+        dimensions=["instrument", "detector"],
+        isCalibration=True,
+    )
+
+    defects = cT.PrerequisiteInput(
```

Comment:
Does this still need to be a prerequisite?

Reply:
Demoted to {{Input}}.